service_management/test/resources/evolutions/test/2.sql

```diff
@@ -0,0 +1,34 @@
+# --- !Ups
+
+CREATE SCHEMA IF NOT EXISTS MUSARK_LOAN;
+
+CREATE TABLE MUSARK_LOAN.LOAN_EVENT (
+  event_id        NUMBER(20) GENERATED BY DEFAULT AS IDENTITY,
+  type_id         INTEGER      NOT NULL,
+  event_date      TIMESTAMP    NOT NULL,
+  registered_by   VARCHAR2(36) NOT NULL,
+  registered_date TIMESTAMP    NOT NULL,
```

Comment:
Since the updated H2 now supports the proper `WITH TIME ZONE` syntax, it's a good idea to add that here as well. A current "hack" that's done when running tests, is to explicitly set the timezone of the JVM to UTC. Adding the `TIME ZONE` property to the `TIMESTAMP` columns _should_ remove the need for that.